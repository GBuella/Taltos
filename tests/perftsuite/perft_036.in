setboard 8/1n4N1/2k5/8/8/5K2/1N4n1/8 w - - 0 1
echo 8/1n4N1/2k5/8/8/5K2/1N4n1/8 w - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
