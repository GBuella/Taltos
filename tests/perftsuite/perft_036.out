8/1n4N1/2k5/8/8/5K2/1N4n1/8 w - - 0 1
14
195
2760
38675
570726
8107539
